common-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '2'
+version: '2.1'
```

Comment:
No objection from me - but curious why the version bump was needed. Something to do with the env var munging below?

Reply:
That's right! Version 2.1 supports including a default value for environment variables: https://docs.docker.com/compose/compose-file/compose-file-v2/#variable-substitution